tests/common/vector_swizzle_assignment.template

```diff
@@ -0,0 +1,56 @@
+/*************************************************************************
+//
+//  SYCL Conformance Test Suite
+//
+//
+//  Copyright (c) 2018-2022 Codeplay Software LTD. All Rights Reserved.
+//  Copyright (c) 2023 The Khronos Group Inc.
+//
+//  Licensed under the Apache License, Version 2.0 (the "License");
+//  you may not use this file except in compliance with the License.
+//  You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+//  Unless required by applicable law or agreed to in writing, software
+//  distributed under the License is distributed on an "AS IS" BASIS,
+//  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+//  See the License for the specific language governing permissions and
+//  limitations under the License.
+//
```

Comment:
While adding new files, you could add a brief description about its role in the header.
For example with some information extracted from the PR description to start with?
Thanks.

Reply:
Added a comment in c034bdf3c936a5f2f828b75eff2798141b1f33b6